docs/index.md

```diff
@@ -9,16 +9,16 @@
 
 ROCm is an open-source software platform optimized to extract HPC and AI workload
 performance from AMD Instinct accelerators and AMD Radeon GPUs while maintaining
-compatibility with industry software frameworks. For more information, see [What is ROCm?](./what-is-rocm.rst)
+compatibility with industry software frameworks. For more information, see
+[What is ROCm?](./what-is-rocm.rst)
 
-If you're using Radeon GPUs, consider reviewing {doc}`Radeon-specific ROCm documentation<radeon:index>`.
-
-Installation instructions are available from:
+The ROCm platform uses the Heterogeneous System Architecture (HSA) to support
+computing across multiple hardware architectures, letting developers write code
+that runs on both AMD CPUs and GPUs. ROCm supports multiple programming
```

Comment:
I know, this has been suggested by @randyh62, but I don't think "implementation details" of the backend are relevant in the introduction. This is information that might not even be relevant to very advanced HIP/ROCm developers, and just bloats the main page.
```suggestion
ROCm supports multiple programming
```